pkg/minikube/cruntime/docker.go

```diff
@@ -106,8 +106,17 @@ func (r *Docker) SocketPath() string {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *Docker) Available() error {
-	_, err := exec.LookPath("docker")
-	return err
+	var err error
+	if _, err = exec.LookPath("docker"); err != nil {
+		return err
+	}
+	if _, err = exec.LookPath("cri-dockerd"); err == nil {
```

Comment:
Also, `cri-dockerd` and `dockerd` only need to be checked if they're using Kubernetes v1.24+, right now it's checking for past Kubernetes versions which don't require them.

Reply:
Yes, now checking for greater than or equal to k8s v1.24 per https://github.com/kubernetes/minikube/issues/14460#issuecomment-1169575735